apps/passport-server/src/services/issuanceService.ts

```diff
@@ -887,14 +1067,133 @@ export class IssuanceService {
       return tickets;
     });
   }
+
+  /**
+   * Verifies a ticket based on:
+   * 1) verification of the PCD (that it is correctly formed, with a proof
+   *    matching the claim)
+   * 2) whether the ticket matches the ticket types known to us, e.g. Zuzalu
+   *    or Zuconnect tickets
+   *
+   * Not used for Devconnect tickets, which have a separate check-in flow.
+   */
+  private async verifyTicket(
```

Comment:
worth renaming this to describe which tickets it can check

Reply:
It *can* check any kind of ticket. The front-end will avoid using this endpoint if it knows it has a Devconnect ticket, but will use it for all others.